### PR TITLE
[basic.fundamental] Redirect some index entries.

### PR DESCRIPTION
**Pull request by Eelis**
Only the index is affected.

### CODE REVIEW

File: source/basic.tex [hunk 1]
```diff
@@ -4845,11 +4845,11 @@
 ``\tcode{\keyword{unsigned} \keyword{char}}'', ``\tcode{\keyword{unsigned} \keyword{short} \keyword{int}}'',
 ``\tcode{\keyword{unsigned} \keyword{int}}'', ``\tcode{\keyword{unsigned} \keyword{long} \keyword{int}}'', and
 ``\tcode{\keyword{unsigned} \keyword{long} \keyword{long} \keyword{int}}''.
-\indextext{type!extended unsigned integer}%
+\indextext{extended unsigned integer type|see{type, extended unsigned integer}}%
 Likewise, for each of the extended signed integer types,
-there exists a corresponding \defn{extended unsigned integer type}.
+there exists a corresponding \defnx{extended unsigned integer type}{type!extended unsigned integer}.
```
Comment (jensmaurer) [inline source/basic.tex:4850]: Make this `\defnadj{extended unsigned integer}{type}` and you get the "see type, extended unsigned integer" entry for free (the other line can go).

**Review by jensmaurer — CHANGES_REQUESTED** [review]
Mostly harmless.

**Comment by Eelis** [comment]
Amended to use \defnadj

**Review by jensmaurer — APPROVED** [review]
Thanks, much shorter = better now.

**Pull request merged**